.gitignore

```diff
@@ -9,4 +9,5 @@ include/*
 lib/*
 templates/*
 mod/*
+!mod/schemes/rootless
```

Comment:
wdyt about making a `vendor/mod` dir?

Reply:
Hadn't thought of that but it definitely makes sense. Only concern would be that we'd have to add a separate line to attempt to include from there as well as the user mod directory.

Reply:
That seems normal and aligned with other patterns (`include`, `lib`)

Reply:
Modified `__mod` to check both the vendor and non-vendor mod dirs for the specified package scheme. Removed the `schemes/` prefix for simplicity.